analytics/tests/test_counts.py

```diff
@@ -1364,24 +1364,46 @@ def assertInviteCountEquals(count: int) -> None:
 
         user = self.create_user(email="first@domain.tld")
         stream, _ = self.create_stream_with_recipient()
-        do_invite_users(user, ["user1@domain.tld", "user2@domain.tld"], [stream])
+
+        invite_expires_in_days = 2
+        do_invite_users(
+            user,
+            ["user1@domain.tld", "user2@domain.tld"],
+            [stream],
+            invite_expires_in_days,
+        )
```

Comment:
I think the same comment about interface here applies -- `invite_expires_in_days` should be a kwargs-only parameter, and then you don't need to define a variable above for readability.